.github/workflows/reusable_guidelines_enforcer.yml

```diff
@@ -0,0 +1,30 @@
+name: Ensure app compliance with Ledger guidelines
+
+on:
+  workflow_call:
+
+jobs:
+  call_check_icons:
+    name: Dispatch
+    uses: ./.github/workflows/_check_icons.yml
+    with:
+      app-repository: ${{ github.repository }}
```

Comment:
Could you define default values in subworkflows (`_check_icons`, `check_makefile`, ...) with `${{ github.repository }}` and `${{ github.event.repository_name }}` which are used in 80% of the time?

Reply:
Done